react/components/FixedContent.js

```diff
@@ -1,45 +1,50 @@
 import React, { Fragment, useState } from 'react'
-import PropTypes from 'prop-types'
 import { ExtensionPoint } from 'vtex.render-runtime'
+import { Container } from 'vtex.store-components'
+import classNames from 'classnames'
 import Logo from './Logo'
 import SearchBar from './SearchBar'
 import Icons from './Icons'
-import { CONSTANTS } from './Helpers'
-import { Container } from 'vtex.store-components'
-import header from '../store-header.css'
 import useDevice from '../hooks/useDevice'
-import classNames from 'classnames'
+import { logo, icons, lean, searchBar, login } from '../defaults'
+
+import styles from '../store-header.css'
 
 /**
- * Component that deals with content thats always fixed on top.
- * Also handles the toggle between mobile search and mobile navbar.
+ * Component that deals with content thats always fixed on top
+ * Also handles the toggle between mobile search and mobile navbar
+ * @param {Boolean} leanMode - if it's leanMode
```

Comment:
why do we need this docs if we already have them on the prop types?

Reply:
Isn't helpful? The idea is that the developer knows instantly what everything means, with no need to visit propTypes. 🤔